### PR TITLE
:sparkles: left column as drawer

### PR DESCRIPTION
**Pull request by antoineeudes**

### CODE REVIEW

File: src/scss/global.scss [hunk 1]
```diff
@@ -385,21 +385,25 @@ h4 {
...
     overflow: auto;
-    height: 100%;
+    position: absolute;
+    top: 40px;
+    z-index: 1;
   }
 
   .left-column.expanded {
     width: 300px;
+    height: 450px;
```
Comment (mathieud) [inline src/scss/global.scss:406]: This should be allowed to grow based on the content size

File: src/components/organisms/InformationLeftColumn/InformationLeftColumn.scss [hunk 2]
```diff
@@ -2,13 +2,21 @@
...
   flex: 0 0 auto;
+  position: relative;
 
   .band-logo {
     max-width: 100%;
     max-height: 100%;
 
     &.expanded {
-      width: 200px;
+      width: 100px;
```
Comment (mathieud) [inline src/components/organisms/InformationLeftColumn/InformationLeftColumn.scss:12]: It's smaller when expanded?

File: src/components/venues/Jazzbar/JazzBarSkeletonPage/JazzBarSkeletonPage.tsx [hunk 3]
```diff
@@ -19,7 +19,7 @@ const JazzBarSkeletonPage: React.FunctionComponent<PropsType> = ({
   return (
     <>
       <WithNavigationBar>
-        <div className="full-page-container experience-container">
+        <div className="full-page-container experience-container jazz-bar-container">
```
Comment (mathieud) [inline src/components/venues/Jazzbar/JazzBarSkeletonPage/JazzBarSkeletonPage.tsx:22]: Can we reduce (group) the classes here? Not sure we need all 3.

File: src/components/organisms/InformationLeftColumn/InformationLeftColumn.scss (hunk 2 above)
Comment (antoineeudes) [reply src/components/organisms/InformationLeftColumn/InformationLeftColumn.scss:12]: smaller than before but still bigger than when not expanded

File: src/scss/global.scss (hunk 1 above)
Comment (antoineeudes) [reply src/scss/global.scss:406]: the div is scrollable. On the mock-ups, it doesn't take full height. Full height doesn't look good at all in my opinion

**Pull request merged**